src/lib/holocene/tab/tab.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  import { goto } from '$app/navigation';
+  import { getContext } from 'svelte';
+  import type { HTMLButtonAttributes } from 'svelte/elements';
+  import { type TabContext, TABS } from './tabs.svelte';
+
+  interface $$Props extends HTMLButtonAttributes {
+    label: string;
+    id: string;
+    'data-testid'?: string;
+    href?: string;
+    panelId?: string;
+    disabled?: boolean;
+    active?: boolean;
+  }
+
+  export let label: string;
+  export let id: string;
+  export let href = '';
+  export let panelId = null;
+  export let disabled = false;
+  export let active = false;
+
+  const { registerTab, selectTab, activeTab } = getContext<TabContext>(TABS);
+
+  registerTab(id);
+
+  $: isActive = href ? active : $activeTab === id;
+
+  const handleClick = () => {
+    selectTab(id);
+    if (href) goto(href);
```

Comment:
Wondering how people feel about using `goto` in a holocene component. I think I prefer it over a component that renders either an `<a />` or a `<button />`, but it still doesn't seem great.

Reply:
I think it's fine to use goto here